kde-misc/kregexpeditor/kregexpeditor-9999.ebuild

```
@@ -4,7 +4,8 @@
 EAPI=8
 
 ECM_HANDBOOK="forceoptional"
-KFMIN=6.3.0
+QTMIN=6.7
+KFMIN=6.5.0
 inherit ecm kde.org
```

```suggestion
KFMIN=6.16.0
QTMIN=6.9.1
inherit ecm kde.org xdg
```

- keep it sorted and use current kde overlay [KDE Gear] 9999 defaults
- xdg: this installs a desktop file ^ and necessary to do if KFMIN >=6.9.0